src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -192,6 +192,11 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
         if (isNaN(withdrawnQuantity)) {
           setIndividualError('withdrawnQuantity', strings.INVALID_VALUE);
           return false;
+        } else {
+          if (withdrawnQuantity > +batches[0].readyQuantity) {
```

Comment:
Will validation still be deferred to the backend if there is more than one batch? Is it possible to validate multiple batches in the frontend?

Reply:
Yes, multiple batches are already validated in a different view (the next step)

Reply:
Charlie, the bulk flow validation happens here https://github.com/terraware/terraware-web/blob/main/src/components/Inventory/withdraw/flow/SelectBatchesWithdrawnQuantity.tsx#L162

Fyi, here's a jira https://terraformation.atlassian.net/browse/SW-2605 I created recently - the single batch flow does not inform user of remaining quantities (except in outplant withdrawal).  We may follow the [input]/remaining pattern soon for single batches (just like we do in bulk and reassignment). My recommendation to design is to also consider using the same flow for both single and bulk. More pros than cons to that.